tests/improved-sanity-test/main.yml

```diff
@@ -481,9 +481,17 @@
     - role: rpm_ostree_install
       packages: httpd
       reboot: false
+      when: ansible_distribution != 'CentOS'
       tags:
         - rpm_ostree_install
 
+    - role: rpm_ostree_install
+      packages: ntp
+      reboot: false
+      when: ansible_distribution == 'CentOS'
```

Comment:
This seems OK, but isn't it more usual Ansible style to derive variables in `vars.yml` and then reference those?

Reply:
Yeah, that sounds right.  I'll fix that.